common/djangoapps/student/helpers.py

```diff
@@ -521,47 +533,40 @@ def _cert_info(user, course_overview, cert_status):
     if status != default_status and course_overview.end_of_course_survey_url is not None:
         status_dict.update({
             'show_survey_button': True,
-            'survey_url': process_survey_link(course_overview.end_of_course_survey_url, user)})
+            'survey_url': process_survey_link(course_overview.end_of_course_survey_url, user),
+        })
     else:
         status_dict['show_survey_button'] = False
 
     if status == 'downloadable':
+        linkedin_config = LinkedInAddToProfileConfiguration.current()
+        share_settings = configuration_helpers.get_value("SOCIAL_SHARING_SETTINGS", settings.SOCIAL_SHARING_SETTINGS)
+        linkedin_share_enabled = share_settings.get('CERTIFICATE_LINKEDIN', linkedin_config.enabled)
```

Comment:
This chunk of code checking the two values feels like it should be a method on LinkedInAddToProfileConfiguration or something. This is contextual logic about what being enabled means that we don't want to push out to every consumer that wants to ask the question.

Reply:
Mentioned above, but yeah, I'm happy to pull inside the model and do it that way.